src/index.js

```diff
@@ -200,6 +204,85 @@ class IpfsRepo {
   exists (callback) {
     this.version.exists(callback)
   }
+
+  /**
+   * Get repo status.
+   *
+   * @param {Object}  options
+   * @param {Boolean} options.human
+   * @param {function(Error, Object)} callback
+   * @return {void}
+   */
+  stat (options, callback) {
+    if (typeof options === 'function') {
+      callback = options
+      options = {}
+    }
+
+    options = Object.assign({}, {human: false}, options)
+
+    parallel([
+      (cb) => this.config.get('Datastore.StorageMax', (err, max) => {
+        if (err) {
+          cb(null, new Big(noLimit))
+        } else {
+          cb(null, new Big(max))
+        }
+      }),
+      (cb) => this.version.get(cb),
+      (cb) => this.blocks.query({}, (err, list) => {
+        list = list || []
+
+        const count = new Big(list.length)
+        let size = new Big(0)
+
+        list.forEach(block => {
+          size = size
+            .plus(block.value.byteLength)
+            .plus(block.key._buf.byteLength)
+        })
+
+        cb(err, {
+          count: count,
+          size: size
+        })
+      }),
+      (cb) => getSize(this.datastore, (err, size) => {
+        cb(err, new Big(size))
```

Comment:
this can be simplified as getSize already returns a bigint

Reply:
Done!